workbench/electron-builder-config.js

```diff
@@ -46,7 +46,7 @@ const config = {
   },
   files: [
     'build/**/*',
-    'node_modules/**/*',
+    'node_modules/**/*', // TODO: probably should exclude any used by renderer.
```

Comment:
Oh I forgot about this one. I think just to avoid duplication since some of these modules like react will have already been bundled by the vite build process. I'll take a stab at this now.

Reply:
Okay I think I accomplished this. It's a little awkward, but electron-builder already knows to exclude anything listed in `package.json` `devDependencies`. So I moved all the front-end deps to that section. They are required for production, but vite already bundles them and puts them in `build/*` so they get packaged that way already. I added a readme section to explain this.

Reply:
Thanks Dave.